dianna/utils/maskers.py

```diff
@@ -1,45 +1,58 @@
+import heapq
 import warnings
 from typing import Union
 import numpy as np
+from numpy import ndarray
 from skimage.transform import resize
 
 
-def generate_masks(input_data: np.array,
-                   number_of_masks: int,
-                   p_keep: float = 0.5):
+def generate_masks(
+    input_data: np.array,
+    number_of_masks: int,
+    feature_res: int = 8,
+    p_keep: float = 0.5,
+):
     """Generate masks for time series data given a probability of keeping any time step or channel unmasked.
 
     Args:
         input_data: Timeseries data to be explained.
         number_of_masks: Number of masks to generate.
         p_keep: the probability that any value remains unmasked.
+        feature_res: Resolution of features in masks.
 
     Returns:
     Single array containing all masks where the first dimension represents the batch.
     """
     if input_data.shape[-1] == 1:  # univariate data
-        return generate_time_step_masks(input_data, number_of_masks, p_keep)
+        return generate_time_step_masks(input_data,
+                                        number_of_masks,
+                                        p_keep,
+                                        number_of_features=feature_res)
 
     number_of_channel_masks = number_of_masks // 3
```

Comment:
can you please add an inline comment and explain `3` in `number_of_masks // 3`?

Reply:
I added a comment.